qutebrowser/browser/commands.py

```diff
@@ -1138,22 +1138,62 @@ def toggle_inspector(self):
                 cur.inspector.show()
 
     @cmdutils.register(instance='command-dispatcher', scope='window')
-    def download(self, url=None, dest=None):
+    def download(self, url=None, dest_old=None, *, mhtml_=False, dest=None):
         """Download a given URL, or current page if no URL given.
 
+        The form `:download [url] [dest]` is deprecated, use `:download --dest
+        [dest] [url]` instead.
+
         Args:
             url: The URL to download. If not given, download the current page.
+            dest_old: (deprecated) Same as dest.
             dest: The file path to write the download to, or None to ask.
+            mhtml_: Download the current page and all assets as mhtml file.
         """
+        if dest_old is not None:
+            message.warning('current', ":download [url] [dest] is deprecated -"
```

Comment:
Instead of `'current'` (which can be problematic in some cases and should be avoided), please use `self._win_id` here.